### DISCUSSION TITLE
Github won't send email verification email.

### DISCUSSION THREAD

**Discussion by aayu22809** [community/community#52560] [GitHub Education]
I am enrolled at SJCC and was trying to connect my email, studentid@stu.sjcc.edu, but it dosnt send the verification email and I think its truncating the .edu part which isnt a real email. How would I connect my email now?

**Comment by samus-aran** [comment]
Hi @aayushyap89 👋 

I'm not sure what you are referring to. We support a number of email addresses successfully but often we hear from students is that their internal mail restrictions block external mail (including GitHub's verification). I would recommend you try again and if you continue to have issues with verifying your email to contact your institutions IT department to potentially add GitHub as a whilelist domain. 👍

**Reply by Zq-502134** [reply]
Hello @samus-aran 
The same situation happened to me, but what I didn't understand was that I received an email saying that I had failed to authenticate. What am I going to do now?

**Reply by Zq-502134** [reply]
oh，surprise！I got the email just now (●'◡'●).
